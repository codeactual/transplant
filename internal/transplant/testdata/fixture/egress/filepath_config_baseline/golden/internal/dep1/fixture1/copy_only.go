package fixture1

const RewrittenImportPath = "copy.tld/user/proj/internal/dep1"
