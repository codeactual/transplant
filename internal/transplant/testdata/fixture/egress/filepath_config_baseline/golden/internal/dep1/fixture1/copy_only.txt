copy.tld/user/proj/internal/dep1
